src/payment/strategies/checkoutcom/checkoutcom-googlepay-payment-strategy.ts

```diff
@@ -0,0 +1,128 @@
+import { CheckoutActionCreator, CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { InvalidArgumentError } from '../../../common/error/errors';
+import { ContentType } from '../../../common/http-request/';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import Payment from '../../payment';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategyActionCreator from '../../payment-strategy-action-creator';
+import { GooglePayPaymentProcessor, GooglePayPaymentStrategy } from '../googlepay';
+
+import { CheckoutcomGooglePayToken, CheckoutcomToken } from './checkout';
+
+export default class CheckoutcomGooglepayPaymentStrategy extends GooglePayPaymentStrategy {
```

Comment:
For consistency, can we name it `CheckoutcomGooglePayPaymentStrategy `

Reply:
renamed as suggested